### ISSUE TITLE
Use Ctrl-P / Ctrl-N to navigate normal intellisense windows

### ISSUE THREAD

**Issue by jaredpar** [jaredpar/VsVim#1149]
Right now Ctrl-P and Ctrl-N can only be used to navigate the windows that VsVim brings up.  It can't navigate normal VS intellisense windows

**Comment by luoxz** [comment]
inoremap `<C-p>` `<up>`
inoremap `<C-n>` `<down>`
set failure，Not working(Navigating intellisense , Ctrl+N and Ctrl-P)

**Comment by chaselal** [comment]
I used to have VS commands bound to CTRL-N and CTRL-P that provides this capability. I forget the names, but they included Emacs. Something like Emacs_prevline IIRC. They gave me up and down line navigation in editors, and I could also navigate intellisense with the same keys. Of course this occupies the keys during normal editing, not just when intellisense is open. Note I was using ReSharper at the time, so that may have affected things.

**Comment by networm** [comment]
I found a solution in [visual studio 2012 - Use C-n/C-p in VsVim to select completion candidate in intellisense popup - Stack Overflow](https://stackoverflow.com/questions/17979779/use-c-n-c-p-in-vsvim-to-select-completion-candidate-in-intellisense-popup).
Just bind ctrl+n to `Edit.LineDown` and ctrl+p to `Edit.LineUp` in Visual Studio settings to solve this problem.
And it also works in ReSharper. I think it is the easiest solution and has no side effect.
Tested on

- `Visual Studio Community 2017 15.4.0`
- `VsVim 2.4.1.0`
- `ReSharper Ultimate 2017.2.2`
